packages/stencil-component/src/components/read-along-component/readme.md

```diff
@@ -7,18 +7,21 @@
 
 ## Properties
```

Comment:
The table in this readme file duplicates that in the readme three levels up. What's the intent of this readme vs that readme? And is the table updated in some automated way in both or do they have to be manually fixed when we add new properties and their associated attributes?

Reply:
one of them is automatically generated, then presumably we copy it into the (manually generated) main readme